demo/src/screens/componentScreens/SegmentedControlScreen.tsx

```diff
@@ -54,17 +78,19 @@ const SegmentedControlScreen = () => {
             inactiveColor={Colors.$textDisabled}
             style={styles.customStyle}
             segmentsStyle={styles.customSegmentsStyle}
+            preset={screenPreset}
```

Comment:
Any meaning for this here when it has a custom style?

Reply:
I removed the preset from this example and moved it to the bottom.